lib/createMessage/fromDiscord.discord.js

```diff
@@ -1,6 +1,7 @@
+const handleEmojis = require('../discord/handleEmojis')
 module.exports = message => {
   return {
-    body: message.content,
+    body: handleEmojis(message.content),
```

Comment:
When receiving a message from Discord -> Discord, the Emoji does not parse at all, this is because it seems to be plaintext? However this current implementation still shows plaintext which makes me believe I might have to have the bot manually add in <:haha:1234567890>